runtime/src/lib.rs

```diff
@@ -329,6 +329,19 @@ impl pallet_sudo::Config for Runtime {
     type WeightInfo = weights::pallet_sudo::NHWeight<Runtime>;
 }
 
+parameter_types! {
+    pub const MaxSignatories: u32 = 100;
+}
+impl pallet_multisig::Config for Runtime {
+    type RuntimeEvent = RuntimeEvent;
+    type RuntimeCall = RuntimeCall;
+    type Currency = Balances;
+    type DepositBase = ConstU128<EXISTENTIAL_DEPOSIT>;
```

Comment:
To be discussed: 
DepositBase -> The base amount of currency needed to reserve for creating a multisig execution or to store a dispatch call for later. set as EXISTENTIAL_DEPOSIT for now

Reply:
I prefer to define a new constant like `BASE_MULTISIG_DEPOSIT` to avoid undesiderable side effects.

Reply:
You can use the same pattern used in polkadot and kusama:


define
```rust
parameter_types! {
	// One storage item; key size is 32; value is size 4+4+16+32 bytes = 56 bytes.
	pub const MultisigDepositBase: Balance = deposit(1, 88);
	// Additional storage item size of 32 bytes.
	pub const MultisigDepositFactor: Balance = deposit(0, 32);
	pub const MaxSignatories: u32 = 100;
}
```

Where the function `deposit` is defined in constant crate and seams to capture the storage cost in some sense:

```rust
	pub const fn deposit(items: u32, bytes: u32) -> Balance {
		items as Balance * 2_000 * CENTS + (bytes as Balance) * 100 * MILLICENTS
	}
```

Reply:
Ok, after some research I think I understand more or less the logic in that computation (deposit method = the cost of storage based on the number of items (storage entries) and the size of those items in bytes)

I changed to these BUT the result are in wei format (or whoever is it called in Substrate) and based on what I saw how value are used in the implementation (existential deposit and tests ) it looks like we use whole numbers.

In order it be consistent I added an additional 

`pub const MultisigDepositBase: Balance = deposit(1, 88) / ACME;`
`pub const MultisigDepositFactor: Balance = deposit(0, 32) / ACME;`

given that `ACME = 1*10^18`

Reply:
I understand the problem and your solution, but I'm not quite sure this is the best approach.
As it stands now, the deposit for multisig operations is almost dust, and that might not be what we want.
I would suggest instead to remove the dividers from the runtime constants, and adjust test balances to be aligned with real world cases. For example, you could multiply each test balance (for all the tests, not only the ones in this PR) by `ACME`?

Reply:
My advice here is to remove `/ ACME` part and mutiply in `testfixtures.rs` file the `starting_balance` fields by `ACME`, leave just the last untouched... in the meantime you can remove the hardcoded `EXISTENTIAL_DEPOSIT` and import it from `crate::EXISTENTIAL_DEPOSIT`.

Reply:
following the mentioned approach